source/agora/flash/Config.d

```diff
@@ -73,6 +73,10 @@ public struct FlashConfig
 
     /// Multiplier for the truncating exponential backoff retrying algorithm
     public uint retry_multiplier = 10;
+
+    /// The seed to use for the keypair of this node
+    @Converter!KeyPair((value) => KeyPair.fromSeed(SecretKey.fromString(value.as!string)))
+    public @Name("seed") immutable KeyPair key_pair;
```

Comment:
Could you add the field to the example config, especially considering that it's a required parameter.
Also document what this key is for (only paying fees).